src/CMakeLists.txt

```diff
@@ -329,6 +329,10 @@ target_link_libraries(
   ZLIB::ZLIB
   qt5keychain)
 
+if(CMAKE_SYSTEM_NAME STREQUAL "FreeBSD")
+  target_link_libraries(mudlet sysinfo)
```

Comment:
Is this library addition part of the qtkeychain change?

Reply:
No, it was something I needed to go in anyhow and I wanted something to force a rebuild on the GH Actions Ubuntu platform as I couldn't see any other way to force a rerun/rebuild.  I am not sure but it is possible that Vadim might have plans to put a button on the Actions user interface that can do it but that is not present here, now {see https://www.actionspanel.app/ }.

Reply:
It is actually the CMake equivalent to #3656 .

Reply:
Also I didn't spot the reason that the PR wasn't merge-able was peer-review and not broken CI.

Reply:
I'm not sure we can have a button to force a rebuild, besides the one that Github shows if a build failed: https://help.github.com/en/actions/configuring-and-managing-workflows/managing-a-workflow-run#viewing-your-workflow-history see "Re-run all checks"

Reply:
Just found that - it wasn't quite where I was expecting - or at least I think it wasn't before.

Reply:
It doesn't work!
![Screenshot_20200424_prior_to_clicking_restart](https://user-images.githubusercontent.com/6163092/80173536-1174de00-85e8-11ea-98a2-2c59a714378e.png)

You can click it and the dark part of the screen (containing the log) goes blank apart from a flashing octo-cat head and you get a green banner at the top saying that the job restarted
![Screenshot_20200424_after_clicking_restart](https://user-images.githubusercontent.com/6163092/80173535-1174de00-85e8-11ea-97ae-7df93673877f.png)

But the button that you click still says "click to restart job".
![Screenshot_20200424_30sec_or_more_after_clicking_restart](https://user-images.githubusercontent.com/6163092/80173534-10dc4780-85e8-11ea-91e8-20462141f0ae.png)

After 30 seconds of no change I clicked it again and this time got a red banner at the top saying there was a problem requesting jobs from GH Accounts:
![Screenshot_20200424_after_clicking_restart_AGAIN](https://user-images.githubusercontent.com/6163092/80173532-1043b100-85e8-11ea-88f5-a839ffbbeae9.png)

Shortly after that I finally did a refresh of the browser tab - and then found that there looked to have been no change - it was still reporting the details from many hours ago:
![Screenshot_20200424_after_refreshing_screen](https://user-images.githubusercontent.com/6163092/80173530-0fab1a80-85e8-11ea-8fff-a6c028d0eb84.png)

Reply:
Interesting to note that the coloured logo initially showed 2/3 green and 1/3 red - which suggests that two runs of the MacOs job had completed by then and one Utubuntu failure but by the end it had changed to 1/4 red (1x failed Ubuntu job), 1/4 orange (1x still in progress Ubuntu) and 1/2 green (2x good MacOS) and the button is now RED saying cancel workflow - and it only did that AFTER I refreshed the screen:
![image](https://user-images.githubusercontent.com/6163092/80174283-18045500-85ea-11ea-8a00-dc2c973e7f8d.png)

Reply:
Please remove this change then, so we keep PRs single topic :slight_smile:

Reply:
Done, BTW.